cmd/query/app/handler_archive_test.go

```diff
@@ -29,13 +29,13 @@ import (
 
 func TestGetArchivedTrace_NotFound(t *testing.T) {
 	mockReader := &spanstoremocks.Reader{}
-	mockReader.On("GetTrace", mock.AnythingOfType("model.TraceID")).
+	mockReader.On("GetTrace", mock.AnythingOfType("*context.valueCtx"), mock.AnythingOfType("model.TraceID")).
```

Comment:
What is `context.valueCtx`? Shouldn't it be `context.Context`?

Reply:
It's an implementation of `context.Context`, I used the implementation because mock.AnythingOfType doesn't appear to work with interfaces. (See https://github.com/stretchr/testify/issues/519)